### DISCUSSION TITLE
Option to turn off internal speaker.

### DISCUSSION THREAD

**Discussion by terenceang** [UzixLS/zx-sizif-512#6] [Ideas]
Thank you for your excellent design; I built the zx sizif D1 and added internal speakers.
 The internal speaker runs off an amplifier board with a mute pin. Can we have a menu option to toggle that pin thru one of the TP7 pins?
 
Thanks. 

https://www.facebook.com/groups/880860392863057/permalink/915995739349522/

**Comment by UzixLS** [comment]
Try this version (I didnt tested, but should work):
[sizif512revd-20210927-speaker.zip](https://github.com/UzixLS/zx-sizif-512/files/7238731/sizif512revd-20210927-speaker.zip)
Pls note that these changes will not be added to mainstream version as they are very specific.

Speaker on TP4 (IO21)

**Reply by terenceang** [reply]
Thank you for the fast response.

Can I request the source file of this change for learning purposes?  

Thank you.

**Reply by UzixLS** [reply]
Of course. Diff file: [speaker.zip](https://github.com/UzixLS/zx-sizif-512/files/7241235/speaker.zip)

**Reply by terenceang** [reply]
Thanks a mil.
